src/style.css

```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+}
```

Comment:
you shouldn't reset indentations in the * selector

Reply:
not fixed! Move this to body selector 
![image](https://user-images.githubusercontent.com/70009205/235709155-9c816bed-759d-49d5-a22b-979833c2f908.png)